src/games/strategy/triplea/delegate/Matches.java

```diff
@@ -121,6 +121,13 @@ public boolean match(final Unit unit) {
       return (ua.getIsCombatTransport() && ua.getIsSea());
     }
   };
+  public static final Match<Unit> UnitIsDefenselessTransport = new Match<Unit>() {
+    @Override
+    public boolean match(final Unit unit) {
+      final UnitAttachment ua = UnitAttachment.get(unit.getType());
+      return (ua.getTransportCapacity() != -1 && ua.getIsSea() && ua.getRawDefense() == 0);
```

Comment:
This exists in v3? How do I find it? 

V3 and v4 don't seem to have any particular written down points of truth which is a bit frustrating.